tracer/src/Datadog.Trace.Tools.Runner/RunSettings.cs

```diff
@@ -3,18 +3,44 @@
 // This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
 // </copyright>
 
-using System.ComponentModel;
-using Spectre.Console.Cli;
+using System.CommandLine;
+using System.CommandLine.Parsing;
 
 namespace Datadog.Trace.Tools.Runner
 {
     internal class RunSettings : CommonTracerSettings
     {
-        [CommandArgument(0, "[command]")]
-        public string[] Command { get; set; }
+        public RunSettings(Command command)
+            : base(command)
+        {
+            command.AddArgument(Command);
+            command.AddOption(AdditionalEnvironmentVariables);
+            command.AddValidator(Validate);
+        }
 
-        [Description("Sets environment variables for the target command.")]
-        [CommandOption("--set-env <VARIABLES>")]
-        public string[] AdditionalEnvironmentVariables { get; set; }
+        public Argument<string[]> Command { get; } = new("command", CommandLineHelpers.ParseArrayArgument, isDefault: false);
```

Comment:
Normally you used `private readonly` fields, but here you have public properties. Any reason?

Reply:
Normally I put the settings directly in the command, so they can be private. Here, those settings are in a different class because they are shared by two commands.